src/app/shell/details/details-tabs/reviews/reviews.component.ts

```diff
@@ -85,14 +92,17 @@ export class ReviewsComponent implements OnInit, OnDestroy {
         takeUntil(this.destroy$),
       ).subscribe((rating: Rate[]) => {
         this.rating = rating;
-        this.isRated = rating?.some((rate: Rate) => rate.parentId === this.parent?.id);
       });
   }
 
   private checkIfAllowedToReview(id: string): void {
     this.store.dispatch(new GetStatusAllowedToReview(this.parent.id, this.workshop.id));
   }
 
+  private checkIfReviewed(id: string): void {
+    this.store.dispatch(new GetReviewedApplications(this.parent.id, this.workshop.id));
+  }
+
```

Comment:
also, I see we can get of these additional methods, we can dispatch actions directly in subscriptions